packages/vue-query/vite.config.ts

```diff
@@ -16,6 +16,7 @@ export default defineConfig({
       if (log.includes('Download the Vue Devtools extension')) {
         return false
       }
+      return undefined
```

Comment:
To resolve this error

![image](https://github.com/TanStack/query/assets/61593290/2469599a-4575-459f-ac1c-b2fedd43e768)